addons_daily/utils/telemetry_data.py

```diff
@@ -241,12 +241,18 @@ def get_trend_metrics(addons_expanded, date):
     addons_expanded = addons_expanded.filter(
         "date >= ({} - INTERVAL 1 DAYS)".format(base_date)
     )
+    absolute_dau = addons_expanded.select(F.countDistinct("client_id")).collect()[0][0]
```

Comment:
This could be written a bit cleaner, i.e:

`addons_expanded.select("client_id").distinct().count()`

Reply:
Changes made!